ToolChain/Debugger/Debugger.py

```diff
@@ -97,11 +99,38 @@ def __init__(self, inputFile=None,
             self.symbols = {}
             self.loadSymbols(symbolsFile=symbolsFile)
 
+        if breakpointFile != "" and breakpointFile is not None:
+            self.breakPoints = self.loadBreakpoints(breakpointFile)
+        else:
+            self.breakPoints =[]
+
         # At this point, debugging session is ready to be used
         self.debugLog("Debugging session is ready to be used. Have fun!")
         self.debug(inputFile=inputFile)
         self.tearDownLoggingFacilities()
 
+    def loadBreakpoints(self, breakpointFile):
+        """
+        This will simply load the breakpoints from file if -bp flag was provided.
+        :param breakpointFile: str, path to file storing breakpoints
+        :return:
+        """
+        self.debugLog("Loading breakpoints from file {}".format(breakpointFile, ))
+
+        file = open(breakpointFile, "r")
+        content = file.readlines()
+        file.close()
+        bp = []
+
+        for line in content:
+            if line != "":
+                line = line.strip("\n")
+                bp.append(int(line))  # breakpoints need to be type int for later conversion to hex address
```

Comment:
It might be a good thing to add some validation here. What if the content of the file is not only made of int? (possibility of this raising ValueError)
That would allow us to generate an appropriate error for the users so they know what is going on.